### PR TITLE
Mobile menu display js

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
this is the creating of a mobile menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,28 +17,28 @@
...
         <h3>welcome</h3>
         <nav class="dropdown-menu">
-          <img src="./imges/dropdown.svg" alt="dropdown" />
+          <img class="mobile-menu-icon" src="./imges/dropdown.svg" alt="dropdown" />
         </nav>
       </header>
-      <header class="header-desctop">
+      <header id="mobile-menu" class="header-desctop">
         <ul>
-          <img src="./imges/icons/close-btn-mobile.svg" alt="btn" />
```
Comment (smcommits) [inline index.html:33]: - [x] **The close button is visible on the desktop as well. It allows me to close the menu even on the desktop version. Please fix this.**
Peek 2021-06-22 03-01

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+});
+mobileIconClose.addEventListener('click', () => {
+  document.getElementById('mobile-menu').style.display = 'none';
+});
+
+liLinks.forEach((item) => {
+  item.addEventListener('click', () => {
+    document.getElementById('mobile-menu').style.display = 'none';
+  });
+});
```
Comment (smcommits) [inline js/script.js:16]: **[Optional]**

**You are adding the same event listener thrice. This violates the principle of DRY.**

You can use arrays, create a function, or use event delegation.

File: index.html [hunk 3]
```diff
@@ -17,28 +17,28 @@
...
-          <li><a href="#">Portflio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+          <li><img class="mobile-menu-close-icon" src="./imges/icons/close-btn-mobile.svg" alt="btn" /></li>
+          <li class="li-link"><a class="menu-link" href="#">Portflio</a></li>
+          <li class="li-link"><a class="menu-link" href="#">About</a></li>
+          <li class="li-link"><a class="menu-link" href="#">Contact</a></li>
```
Comment (smcommits) [inline index.html]: - [x] **Please make sure that the links are working.**

Hint: You can give id to the respective elements. Suppose I have an element with `id="something`. I can use `<a href="#something">` to create a link to that element.

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @yahyaelganyni1, 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] **[Optional] You should try add a more descriptive PR description.**

For instance, your PR description can give a summary of changes you made to implement a functioning mobile menu. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi Akasha and Yahya! :wave: :wave:

####  Your project is complete! Nice job done for implementing the changes requested by the previous reviewer(s) :clap:  :clap: Your mobile menu is working correctly and your portfolio looks great! Congratulations 🎉🍾🍾 . There are however a few recommendations to implement to improve your project. Kindly check the optional comment below : 👍

## Good points 👍
No Linters errors ✔️
Semantics and syntax  are well used✔️

#### You may merge, good luck on your next project! :rocket:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**

- [OPTIONAL] The provided [demo link](https://yahyaelganyni1.github.io/mobile-version-skeleton/) in your ReadMe still displays the unrevised project. Kindly ensure to update the same.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**